<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WT_DataWidget</class>
 <widget class="QWidget" name="WT_DataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WT_DataWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 全局设置 */
QWidget {
    background-color: #FFFFFF;
    color: #333333;
    font-family: &quot;Microsoft YaHei&quot;, sans-serif;
    font-size: 13px;
}

/* 顶部操作区：保留背景色，无边框 */
QWidget#topPanel {
    background-color: #FFFFFF;
    border: none;
}

/* 底部信息栏 */
QWidget#bottomPanel {
    background-color: #FAFAFA;
    border-top: 1px solid #E8E8E8;
}

/* 按钮通用样式：蓝色主题 */
QPushButton {
    background-color: #1890FF;
    border: none;
    border-radius: 4px;
    padding: 5px 15px;
    color: #FFFFFF;
    min-height: 24px;
    font-weight: bold;
    font-size: 12px;
}
QPushButton:hover {
    background-color: #40A9FF;
}
QPushButton:pressed {
    background-color: #096DD9;
}
QPushButton:disabled {
    background-color: #F5F5F5;
    color: #CCCCCC;
    border: 1px solid #E8E8E8;
}

/* 特殊按钮：同步保存 (绿色) */
QPushButton#btnSave {
    background-color: #52C41A;
}
QPushButton#btnSave:hover {
    background-color: #73D13D;
}
QPushButton#btnSave:pressed {
    background-color: #389E0D;
}

/* QTabWidget 样式优化：体现分区，显示横线 */
QTabWidget::pane {
    border: none;
/* 这里设置横线，颜色稍深以体现分割 */
    border-top: 1px solid #D9D9D9;
    background: #FFFFFF;
/* top: 0px 表示不重叠，横线完整显示 */
    top: 0px;
}

QTabBar::tab {
    background: #FFFFFF;
    border: none;
/* 底部透明边框，占位 */
    border-bottom: 2px solid transparent;
    padding: 8px 20px;
    min-width: 80px;
    color: #666666;
    margin-right: 2px;
/* 底部外边距为0，紧贴横线 */
    margin-bottom: 0px;
}

QTabBar::tab:selected {
    color: #1890FF;
    font-weight: bold;
/* 选中时，底部出现蓝色指示条，位于横线之上 */
    border-bottom: 2px solid #1890FF;
    background-color: #E6F7FF;
/* 可选：选中时背景微蓝，增加区分度 */
}

QTabBar::tab:hover {
    color: #40A9FF;
}

/* 底部标签 */
QLabel#filePathLabel {
    color: #666666;
    font-weight: bold;
}
QLabel#statusLabel {
    color: #999999;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_Main">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="topPanel" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_Top">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_FileOps">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnOpenFile">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>-1</pointsize>
            <bold>true</bold>
            <hintingpreference>PreferDefaultHinting</hintingpreference>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>打开数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>同步保存</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnExport">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>导出 Excel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_File">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_DataTools">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnDefineColumns">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>定义列属性</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnTimeConvert">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>时间转换</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPressureDropCalc">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>计算压降</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCalcPwf">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>套压转流压</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnErrorCheck">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>异常检查</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_Tools">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <property name="tabsClosable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomPanel" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_Bottom">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QLabel" name="filePathLabel">
        <property name="text">
         <string>未加载文件</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_Bottom">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>就绪</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
